couples.go

```diff
@@ -105,12 +105,13 @@ func (couples *Couples) Consume(deps map[string]interface{}) (map[string]interfa
 
 func (couples *Couples) Finalize() interface{} {
 	peopleMatrix := make([]map[int]int64, couples.PeopleNumber)
-	fmt.Printf("people_files:\n")
+	peopleFiles := make([][]string, couples.PeopleNumber)
```

Comment:
How about keeping track of the number of times a developer changed a file? `[]map[string]int` Just asking